FirefoxLite_Downloader_Library/src/main/java/com/example/firefoxlitedownloaderlibrary/MainFunctionAccessFile.java

```diff
@@ -0,0 +1,25 @@
+package com.example.firefoxlitedownloaderlibrary;
```

Comment:
btw,  I think we don't can use `org.mozilla` instead of `com.example`

Reply:
renamed the the librar

Reply:
@cnevinc WIll make a separate issue for it and send across a pull request for it.

Reply:
@cnevinc  For now marking this resolved to avoid confusion